csrc/fusion.h

```diff
@@ -261,8 +261,16 @@ class NVF_API Fusion : public IrContainer {
   //! aliased.
   const AliasInfo& getOutputAlias(const Val* output) const;
 
+  //! Returns the expressions that require codegen.
+  //! Expressions where all the outputs are marked to be computed through
+  //! expression evaluator are filtered out.
   std::vector<Expr*> getExprsToCodegen();
 
+  //! Returns the fusion outputs that require codegen.
+  //! The fusion outputs to be computed through expression evaluator are
+  //! filtered out.
+  std::vector<Val*> getFusionOutputsRequiringCodegen();
+
```

Comment:
Move these two methods outside the class, i.e., 

```
std::vector<Val*> getExprsToCodegen(Fusion* fusion);
std::vector<Val*> getFusionOutputsRequireingCodegen(Fusion* fusion);
```

Reason: these two functions don't use and shouldn't need to use any private fields of Fusion. Making them class methods allows future writers to misuse private fields, making it harder to remove/change these fields in the future.